d2l-table-shared-styles.html

```diff
@@ -53,7 +53,6 @@
 			};
 			--d2l-table-row: {
 				display:table-row;
-				background-color: var(--d2l-table-body-background-color);
```

Comment:
You should also remove https://github.com/BrightspaceUI/table/pull/179/files#diff-6b62292c087ca636d85d5167970ae85fR11 since I don't think it's used anymore?

Reply:
It might be referenced from the --d2l-table-body and --d2l-table-foot rules.